.github/workflows/kind-e2e.yaml

```diff
@@ -12,8 +12,8 @@ env:
   # https://github.com/google/go-containerregistry/pull/125 allows insecure registry for
   # '*.local' hostnames. This works both for `ko` and our own tag-to-digest resolution logic,
   # thus allowing us to test without bypassing tag-to-digest resolution.
-  KIND_VERSION: 0.14.0
-  KNATIVE_VERSION: 1.6.0
+  KIND_VERSION: 0.20.0
```

Comment:
We should probably use the same kind versions both in this test and [as a minimum install version](https://github.com/knative-sandbox/kn-plugin-quickstart/blob/7549e95ec3beaafe44f2f32ed80c2ca64cba1767/pkg/kind/kind.go#L31), so may want to update the other place too...